### ISSUE TITLE
Renaming of the 'acc' and 'loc' input file names to highlight property

### ISSUE THREAD

**Issue by MattDonovan82** [OasisLMF/OpenDataStandards#49]
As the number of OED input files will increase for other LOBs and humanitarian uses, what is the consensus about changing the name of the current fields to highlight that they are for 'property' business? I suggest something like:

'acc_prop'
'loc_prop'

Would this cause any potential operational issues?

**Comment by johcarter** [comment]
I don't think it would cause any operational issues in oasis as files can be uploaded with any name, but I would vote to keep account and location as they are, and give the new files new names.

**Comment by benhayes21** [comment]
closing
